Documentation/EtcdWatch.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+The EtcdWatch provide methods to watch on a key and cancel a watcher, if the watcher is disconnected on error, it will be resumed automatically.
+
+## requestStream instance
+
+StreamObserver instance
```

Comment:
Why do you have "StreamObserver instance" here?
I'm really confused. You have mostly talked about requestStream below?